helm/goldilocks-app/charts/vpa/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
```

Comment:
Same as the previous comment, I don't think we need to ship it as a sub chart, we should just pull it in